modules/default/clock/clock.js

```diff
@@ -230,13 +230,13 @@ Module.register("clock", {
 			moonWrapper.innerHTML =
 				'<span class="' +
 				(isVisible ? "bright" : "") +
-				'"><i class="fa fa-moon-o" aria-hidden="true"></i> ' +
+				'"><i class="fas fa-moon-o" aria-hidden="true"></i> ' +
```

Comment:
```suggestion
				'"><i class="fas fa-moon" aria-hidden="true"></i> ' +
```
<img width="131" alt="Bildschirmfoto 2022-01-05 um 13 38 27" src="https://user-images.githubusercontent.com/9592452/148219345-f6e942c9-f952-4107-afdc-e1f818ec304b.png">
<img width="150" alt="Bildschirmfoto 2022-01-05 um 13 38 22" src="https://user-images.githubusercontent.com/9592452/148219342-b98ed626-a9e0-403a-a98c-021894dd74a3.png">